configs/server/forseti_conf_server.yaml.in

```diff
@@ -527,16 +527,8 @@ notifier:
     violation:
       cscc:
         enabled: false
-        mode: api
-        # Alpha API
-        organization_id: {ROOT_RESOURCE_ID}
-        # gcs_path should begin with "gs://"
-        gcs_path:
-        # Beta API
-        # Cloud SCC Beta API uses a new source_id.  It is unique per
+        # Cloud SCC uses a source_id. It is unique per
```

Comment:
Can you please confirm or double-check, there is nothing that is looking at the `mode` key anymore?

Reply:
`mode` is not used in code anymore. It's required for Alpha API, but not for Beta and GA. Now that we have deprecated alpha, it is safe to remove it.